### PR TITLE
[5.3.x] Apply validation of code format(java,xml) and Modify code format. #680

### PR DESCRIPTION
**Pull request by btyanagawamg**
Please review #680 .
This PR is backport for 5.3.x .

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -187,5 +188,37 @@
...
+                    <plugins>
+                        <plugin>
+                            <groupId>org.codehaus.mojo</groupId>
+                            <artifactId>xml-maven-plugin</artifactId>
+                            <version>${xml-maven-plugin.version}</version>
+                            <configuration>
+                                <indentSize>2</indentSize>
+                                <formatFileSets>
+                                    <formatFileSet>
+                                        <directory>${project.basedir}</directory>
```
Comment (yoshikawaa) [inline pom.xml]: - [ ] Please check https://github.com/terasolunaorg/terasoluna-gfw/pull/700/files#diff-600376dffeb79835ede4a0b285078036R208 .
`<directory>${project.basedir}/terasoluna-gfw-parent</directory>` vs `<directory>${project.basedir}</directory>`

File: terasoluna-gfw-common-libraries/terasoluna-gfw-web-jsp/src/main/resources/META-INF/functions.tld [hunk 2]
```diff
@@ -109,32 +110,35 @@
...
+    </description>
+    <name>query</name>
+    <function-class>org.terasoluna.gfw.web.el.Functions</function-class>
+    <function-signature>java.lang.String query(java.lang.Object)</function-signature>
+    <example><![CDATA[
 <a href="search?page=2&${f:query(searchForm)}">next page</a>
-        ]]></example>
-    </function>
+        ]]>
+    </example>
```
Comment (yoshikawaa) [inline terasoluna-gfw-common-libraries/terasoluna-gfw-web-jsp/src/main/resources/META-INF/functions.tld]: - [ ] Please divide the commit between the part changed manually and the part changed by formatter.

File: pom.xml (hunk 1 above)
Comment (btyanagawamg) [inline pom.xml]: Sorry . I forgot to reflect the contents of the 2a196186373104334f4882c6e3c4ac5798f0fc05 commit .
I cherry-picked above commit at 6286158d43d78d7c80cb869df11719f295622965 commit .

File: terasoluna-gfw-common-libraries/terasoluna-gfw-web-jsp/src/main/resources/META-INF/functions.tld (hunk 2 above)
Comment (btyanagawamg) [inline terasoluna-gfw-common-libraries/terasoluna-gfw-web-jsp/src/main/resources/META-INF/functions.tld]: I divide below 2 commits .
- 9bc71b4c4f16c828f75d8624ffe14f85de37c938 tld files formatted by formatter
- d2c0105c234ab4f3586f331e1cabd759efbbf641 tld files formatted by manually

**Pull request merged**